dags/libero/xml/libero/__init__.py

```diff
@@ -16,4 +16,4 @@ def get_content_id(jats_xml: ElementTree) -> str:
         xml=jats_xml,
         xpaths=[xpaths.ID],
         namespaces=LIBERO_MAP
-    )
+    )[0]
```

Comment:
could assert this list has 1 element before returning the 0-th one?

Reply:
If I read that correctly, that asserts that the number of elements is `> 0` whereas here we should check that it is `== 1` to avoid silently discarding elements after the first

Reply:
It's likely that there will be multiple results as multiple XPaths (strategies) can be used to derive the target element. As these are ordered, the first match is used as it should be higher in the order of preferred locations.

Reply:
Ah it's intended then, :+1: